src/components/docs-anchor/index.tsx

```diff
@@ -1,60 +0,0 @@
-import Link from 'next/link'
-import classNames from 'classnames'
-import { useCurrentProduct } from 'contexts'
-import InlineLink from 'components/inline-link'
-import s from 'components/inline-link/inline-link.module.css'
-
-const DocsAnchor = ({
-	href,
-	children,
-	...rest
-}: JSX.IntrinsicElements['a']) => {
-	const currentProduct = useCurrentProduct()
-	const passableProps = {
-		...rest,
-		// TODO: we probably want to make use of the InlineLink component in the future instead of just adopting its styles here. Tackle that when we have bandwidth to VQA lots of links.
-		className: classNames(rest.className, s.root),
-	}
-
-	if (!href) {
-		return <a {...passableProps}>{children}</a>
-	}
-
-	// Authors write content as if it only exists for their product,
-	// eg Waypoint content contains links that start with "/docs".
-	// We need to be adjust these links, eg to start with "/waypoint/docs".
-	let adjustedHref = href
-	// TODO: infer this condition dynamically
-	if (
-		currentProduct.basePaths.some((basePath) => href.startsWith(`/${basePath}`))
-	) {
-		adjustedHref = `/${currentProduct.slug}${href}`
-	}
-
-	// Render inline link if it's for sure a string and not a heading permalink
-	if (typeof children === 'string' && children !== '»') {
-		return (
-			<InlineLink href={adjustedHref} textWeight="medium">
-				{children}
-			</InlineLink>
-		)
-	}
```

Comment:
I think the intent here was to only render `InlineLink` for links with string children.

The new `MdxA` component renders `InlinkLink` in this way, so serves the same purpose for this case.